resources/gnomad_liftover.py

```diff
@@ -0,0 +1,160 @@
+import argparse
+from basics import *
+from gnomad_hail import *
+import logging
+
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("gnomAD_liftover")
+logger.setLevel(logging.INFO)
+
+
+def checkpoint(ht: hl.Table, dtype: str) -> hl.Table:
+    """
+    Creates a "checkpoint" by writing Table out and reading back in
+    :param Table ht: Table to be evaluated and saved
+    :param str dtype: Data type (exomes or genomes)
+    :return: Table after writing out and reading back in
+    :rtype: Table
+    """
+
+    if dtype == 'exome':
+        ht.write('gs://seqr-datasets/methods_dev/test_data/kc/gnomAD_exome_sites.ht', overwrite=True)
+        ht = hl.read_table('gs://seqr-datasets/methods_dev/test_data/kc/gnomAD_exome_sites.ht')
+
+    else:
+        ht.write('gs://seqr-datasets/methods_dev/test_data/kc/gnomAD_genome_sites.ht', overwrite=True)
+        ht = hl.read_table('gs://seqr-datasets/methods_dev/test_data/kc/gnomAD_genome_sites.ht')
+
+    return ht
+
+
+def lift_ht(ht: hl.Table, dtype: str, rg: hl.genetics.ReferenceGenome) -> hl.Table:
+    """
+    Lifts gnomAD release table from b37 to b38
+    :param Table ht: gnomAD release Table
+    :param str dtype: Data type (exome or genome)
```

Comment:
Probably worth spelling out `data_type` to be consistent with the rest of the codebase.